.github/workflows/build.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           prefix: /opt
 
       - name: sanitize macOS runners
-        if: matrix.platform.os == 'macos-11'
+        if: fromJson(needs.get-platform.outputs.build-os) == 'macos-11'
```

Comment:
yuck

Reply:
I theoretically agree, but the self-hosted runner identifiers are `string[]` rather than `string`.

Reply:
yeah not complaining at you, more at GitHub for forcing this

Reply:
coders like to code, right? who cares about complexity? ;)